Automattic/jetpack
# Boost: Fix accidental parallel critical CSS regeneration

Fixes #31543

## Proposed changes:

* Hide the regenerate button immediately on click to avoid accidental parallel requests

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
1. Install and setup Boost (free);
2. Wait for Critical CSS to finish generating;
3. Click multiple times on the Regenerate button;
4. If you have the network inspector on XHR requests, you should not see multiple start requests for regeneration.
